src/include/cdb/cdbvars.h

```diff
@@ -722,6 +722,8 @@ extern bool gp_mk_sort_check;
 
 extern bool trace_sort;
 
+extern bool optimize_bounded_sort;
```

Comment:
It is already defined in [guc.c](https://github.com/arenadata/gpdb/blob/6137644204f182612da87430258ad183ac7c8df4/src/backend/utils/misc/guc.c#L149).